src/lib/workspace/sharing/ConfirmPublicAnalysis.tsx

```diff
@@ -72,7 +72,7 @@ export default function ConfirmPublicAnalysis({
                 target="_blank"
                 style={{ color: theme?.palette.primary.hue[600] }}
               >
-                ClinEpiDB Public analyses page.
+                Public analyses page.
```

Comment:
I know the original has a period, but this isn't a sentence and likely doesn't need one.

How does it look on the webpage (could you provide a screenshot pls - I am lazy!)

Reply:
Hi Bob! It is a sentence actually---the sentence says "This analysis will appear on the Public analyses page." Here's the modal:

![image](https://user-images.githubusercontent.com/12377649/184422926-e6a1cb9a-0626-47d0-8249-56dcfdd332e0.png)